test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/010_ATF_P_TC_media_app_PTU_fail_ForceProtectedService_ON_RPC_encrypted_true.lua

```diff
@@ -11,11 +11,10 @@
 -- SDL must respond StartService (NACK) to this mobile app
 --
 -- 1. Used preconditions:
--- RPC SetAudioStreamingIndicator is allowed by policy
 -- ForceProtectedService is set to ON in .ini file
 -- Media app exists in LP, no certificate in module_config
 -- Register and activate application.
--- Send StartService(serviceType = 7 (RPC), RPCfunctionID = 48(SetAudioStreamingIndicator))
+-- Send StartService(serviceType = 7 (RPC))
```

Comment:
@istoimenova, Please remove "SDL should not process request to HMI" statement in line 20